functions/lib/integration/import-order.js

```diff
@@ -103,7 +103,7 @@ module.exports = ({ appSdk, storeId, auth }, tinyToken, queueEntry, appData, can
   if (typeof tinyOrderNumber === 'string' && tinyOrderNumber.startsWith('id:')) {
     job = getTinyOrder(tinyOrderNumber.substring(3))
   } else {
-    job = tiny.post('/pedidos.pesquisa.php', { numero: tinyOrderNumber })
+    job = tiny.post('/pedidos.pesquisa.php', { numeroEcommerce: String(tinyOrderNumber) })
```

Comment:
dar uma acelerada...

```suggestion
    const filter = typeof tinyOrderNumber === 'string' && tinyOrderNumber.startsWith('ecom:')
      ? { numeroEcommerce: String(tinyOrderNumber) }
      : { numero: tinyOrderNumber }
    job = tiny.post('/pedidos.pesquisa.php', { numeroEcommerce: String(tinyOrderNumber) })
```